client/src/components/RatingsComponents/ReviewsList/HelpfulTile.jsx

```diff
@@ -21,13 +24,35 @@ function HelpfulTile ({helpfulnessCount, reviewId}) {
     });
   };
 
+  const handleReported = () => {
+    Axios({
+      method: 'put',
+      url: `https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfp/reviews/${reviewId}/report?review_id=${reviewId}`,
+      headers: {
+        'Authorization': token
+      }
+    })
+    .then(() => console.log('you have reported review # ', reviewId))
+    .catch((err) => console.log('error reporting review', err));
```

Comment:
I would use console.error since it makes it easier to spot the error since it gives a red popup instead of a grayed out console.log statement. It's personal preference though